src/ripples/focus-ripple.jsx

```diff
@@ -57,6 +80,8 @@ const FocusRipple = React.createClass({
   },
 
   _getRippleElement(props) {
+    const theme = this.state.muiTheme.ripple;
```

Comment:
It's intentional. The simpler our component are, the best it's. Here, we can avoid to add an additionnal state and context property and keep our component stateless.